### DISCUSSION TITLE
Where is start?

### DISCUSSION THREAD

**Discussion by sixt00** [microsoft/Web-Dev-For-Beginners#680] [Q&A]
Hi, who can me where is click for start course?

**Comment by jlooper** [comment]
There is no button, please follow the instructions on the README and go through the lessons listed in the table there.

**Comment by sanamhub** [comment]
> Hi, who can me where is click for start course?

Click on this link to go-to lessons:
https://github.com/microsoft/Web-Dev-For-Beginners#lessons

Go through lesson 1, 2, 3 and so on..

![image](https://user-images.githubusercontent.com/31341013/172283737-4ee52e83-f4d2-466c-838a-0cfa764cf7d9.png)
